runtime/dolphin/src/nimbus_session_adapter.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020-2022 Manta Network.
+// This file is part of Manta.
+//
+// Manta is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Manta is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Manta.  If not, see <http://www.gnu.org/licenses/>.
+
+//! This file contains wrappers for Nimbus to make them compatible with pallet session traits,
+//! but has No-Op implementations on everything so as to not change behavior of the running chain just yet
+
+use frame_support::traits::OneSessionHandler;
+use frame_system::Config;
+use pallet_author_inherent::Pallet as AuthorInherent;
+use sp_application_crypto::BoundToRuntimeAppPublic;
+
+/// This adapts pallet AuthorInherent to be compatible with pallet session
+/// making it suitable as a SessionKey entry
+pub struct AuthorInherentWithNoOpSession<T: Config>(pub AuthorInherent<T>);
+
+impl<T: Config> BoundToRuntimeAppPublic for AuthorInherentWithNoOpSession<T> {
+    type Public = <AuthorInherent<T> as BoundToRuntimeAppPublic>::Public;
+}
+
+impl<T: Config> OneSessionHandler<T::AccountId> for AuthorInherentWithNoOpSession<T> {
```

Comment:
in short: yes

purestake does not use the `Session` pallet, we do.
Currently, as nimbus is disabled we need a no-op adapter for their types since we don't want to change them directly.
In nimbus stage 1 these will be exchanged to full handlers.